### PR TITLE
Fix: Word wrap on labels does not work if width and height are not set

### PR DESCRIPTION
**Pull request by infosia**
Fix for [TIMOB-19048](https://jira.appcelerator.org/browse/TIMOB-19048).

```javascript
var win = Ti.UI.createWindow({ backgroundColor: 'white' });

var lbl = Ti.UI.createLabel({
    text: '*** Resources Dir: Test presence of file that exists => blahTest presence of file that does not exist => bleh*** appData Dir:Test presence of file that exists => meh Test presence of file that does not exist => merh',
    color: 'black'
});

win.add(lbl);
win.open();
```

windows_8_1

### CODE REVIEW

File: Source/UI/src/Label.cpp [hunk 1]
```diff
@@ -15,6 +14,32 @@ namespace TitaniumWindows
...
+		WindowsLabelLayoutDelegate::WindowsLabelLayoutDelegate() TITANIUM_NOEXCEPT
+			: WindowsViewLayoutDelegate()
+		{
+		}
+
+		void WindowsLabelLayoutDelegate::setComponent(Windows::UI::Xaml::FrameworkElement^ label) 
+		{
+			WindowsViewLayoutDelegate::setComponent(label);
+
+			// TIMOB-19048: max width is set to screen width by default
```
Comment (sgtcoolguy) [inline Source/UI/src/Label.cpp]: So setting this to the width of the current window seems Ok for the case when the label is basically just a child of the windows (or some view that fills the window). I assume the true max width should likely be the width of the parent component. If the label is part of some view that is expected to be half the size of the window, we don't want the label to extend out past it, do we?

Comment (infosia) [inline Source/UI/src/Label.cpp]: you're right. mmm..

**Comment by infosia** [comment]
```javascript
var win = Ti.UI.createWindow({ backgroundColor: 'white' });

var view = Ti.UI.createView({
    width: '80%',
    height: '80%',
    backgroundColor: 'gray'
});
var lbl = Ti.UI.createLabel({
    text: '*** Resources Dir: Test presence of file that exists => blahTest presence of file that does not exist => bleh*** appData Dir:Test presence of file that exists => meh Test presence of file that does not exist => merh',
    color: 'black'
});

view.add(lbl);
win.add(view);
win.open();
```

windows_8_1

File: Source/UI/src/Label.cpp (hunk 1 above)
Comment (infosia) [inline Source/UI/src/Label.cpp]: should be fixed now.

**Pull request merged**